corehq/apps/cloudcare/models.py

```diff
@@ -68,34 +87,59 @@ def get_by_domain(cls, domain):
         return self or cls(domain=domain)
 
     def user_can_access_app(self, user, app):
-        user_id = user['_id']
         app_id = app['_id']
-        if not self.restrict or user['doc_type'] == 'WebUser':
+        app_ids = (app_id, app['copy_of'] or ())
```

Comment:
I don't know. It was there before, which isn't a good excuse, but I was trying to make as few changes as possible